### PR TITLE
June update

### PR DESCRIPTION
**Pull request by gabornagyfarkas**
Updates with the latest event from June.

### CODE REVIEW

File: presentations/June/jobs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+We are looking for a passionate full-time iOS developer to join our team. Contact me at [vasarhelyi.agnes@ustream.tv](mailto:vasarhelyi.agnes@ustream.tv).
+
+We are looking for passionate full-time iOS/macOS developers to join our team. Contact me at [tamas.lustyik@prezi.com](mailto:tamas.lustyik@prezi.com)
+
+We are looking for mid-level or senior Objective-C iOS developers at Misys! If you want to work on our banking app, join us. The team is quite young, office at Oktogon, free coffee and tea. Bakery day, soup day, beer day and more! If you are interested or have questions, just drop me a mail: [andras.somogy@misys.com](mailto:andras.somogy@misys.com)
```
Comment (vasarhelyia) [inline presentations/June/jobs.md]: address should be andras.somogyi@misys.com

**Comment by vasarhelyia** [comment]
otherwise, awesome! 💯 🎉 👍

**Comment by gabornagyfarkas** [comment]
Fixed the typo and added the links to the videos :)

File: presentations/June/jobs.md [hunk 2]
```diff
@@ -2,6 +2,6 @@ We are looking for a passionate full-time iOS developer to join our team. Contac
 
 We are looking for passionate full-time iOS/macOS developers to join our team. Contact me at [tamas.lustyik@prezi.com](mailto:tamas.lustyik@prezi.com)
 
-We are looking for mid-level or senior Objective-C iOS developers at Misys! If you want to work on our banking app, join us. The team is quite young, office at Oktogon, free coffee and tea. Bakery day, soup day, beer day and more! If you are interested or have questions, just drop me a mail: [andras.somogy@misys.com](mailto:andras.somogy@misys.com)
```
Comment (vasarhelyia) [inline presentations/June/jobs.md]: fix the address in the mailto part as well please

File: presentations/June/jobs.md [hunk 3]
```diff
@@ -2,6 +2,6 @@ We are looking for a passionate full-time iOS developer to join our team. Contac
...
+We are looking for mid-level or senior Objective-C iOS developers at Misys! If you want to work on our banking app, join us. The team is quite young, office at Oktogon, free coffee and tea. Bakery day, soup day, beer day and more! If you are interested or have questions, just drop me a mail: [andras.somogyi@misys.com](mailto:andras.somogy@misys.com)
```
Comment (vasarhelyia) [inline presentations/June/jobs.md]: fix the address in the mailto part as well please

**Comment by gabornagyfarkas** [comment]
Both emails are fixed.

**Pull request merged**